pkg/kds/mux/client.go

```diff
@@ -105,10 +105,9 @@ func (c *client) Start(stop <-chan struct{}) (errs error) {
 	defer cancel()
 
 	log := muxClientLog.WithValues("client-id", c.clientID)
-	errorCh := make(chan error)
+	errorCh := make(chan error, 6)
```

Comment:
Setting the channel capability to 6 to avoid the goroutine leak due to the following 6 goroutines methods `errorCh` sending stuck

Reply:
put the comments in codes in another PR to solve the goroutine leak problem